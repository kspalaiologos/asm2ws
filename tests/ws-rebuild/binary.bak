@S  0 / DUP
    GETN
    RCL
    DUP / BLTZ %O
    STO 0, 1
@N  DUP / MOD 2
    RCL 0 / DUP
    ADD 1
    0 / XCHG / STO
    XCHG / STO
    DIV 2
    DUP / JZ %P
    JMP %N
@P  0 / DUP
    RCL / SUB 1
    REP DUP 2
    JZ %Q
    RCL / PUTN
    STO
    JMP %P
@Q  PUTC 10 / JMP %S
@O  END

