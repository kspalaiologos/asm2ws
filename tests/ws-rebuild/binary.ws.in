1237
59129
691237
60010
28517
239501
-1
