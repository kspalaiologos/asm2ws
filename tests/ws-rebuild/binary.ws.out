10011010101
1110011011111001
10101000110000100101
1110101001101010
110111101100101
111010011110001101
